### PR TITLE
initial cublaslt support for MLP

### PR DESCRIPTION
**Pull request by seryilmaz**
Adds cublasLt support for MLP

### CODE REVIEW

**Review by cliffwoolley — CHANGES_REQUESTED** [review]
Please add the correct license headers to these files.

**Comment by cliffwoolley** [comment]
@matthew-frank please review.

File: csrc/mlp.cpp [hunk 1]
```diff
@@ -60,9 +90,11 @@ std::vector<at::Tensor> mlp_forward(int use_bias, int activation, std::vector<at
   auto reserved_size = get_mlp_reserved_space(batch_size, num_layers, output_features.data());
 
   // create output/workspace tensor
-  // TODO(deyuf): just get buffer?
+  // TODO(deyuf): just get buffer? why empty with .type() gets GPU tensor?
```
Comment (matthew-frank) [inline csrc/mlp.cpp:93]: is there an open TODO here?  If so, what?  If not, remove the TODO?

Comment (seryilmaz) [reply csrc/mlp.cpp]: removed now.

File: csrc/mlp.cpp [hunk 2]
```diff
@@ -1,10 +1,39 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
```
Comment (ptrblck) [inline csrc/mlp.cpp:2]: Should this be changed to 2021?

**Review by matthew-frank — APPROVED** [review]

File: csrc/mlp.cpp (hunk 2 above)
Comment (cliffwoolley) [reply csrc/mlp.cpp:2]: At least add the -2021.  We can come back later and delete the 2020- part if it's inaccurate.

**Review by cliffwoolley — APPROVED** [review]

**Comment by LSinev** [comment]
@seryilmaz Please consider restoring support for cuda 10.0 which do not have cublaslt.
Installation instruction
```
pip install -v --disable-pip-version-check --no-cache-dir --global-option="--cpp_ext" --global-option="--cuda_ext" ./
```
is not working now with cuda 10.0, and removing `--cuda_ext` turns off ability to use FusedNovograd which was usable before this improvement
```
RuntimeError: apex.optimizers.FusedNovoGrad requires cuda extensions
```

**Pull request merged**